submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/repositories/Repositories.kt

```diff
@@ -18,8 +18,8 @@ import org.springframework.data.mongodb.repository.Query
 import java.util.stream.Stream
 
 interface SubmissionMongoRepository : MongoRepository<DocSubmission, ObjectId> {
-    @Query(value = "{ 'accNo': '?0', 'version': { \$gte: 0 } }", fields = "{ relPath : 1 }")
-    fun getRelPath(accNo: String): SubmissionRelPath
+    @Query(value = "{ 'accNo': '?0', 'version': { \$gte: 0 } }", fields = "{ relPath : 1, released: 2 }")
+    fun getSubData(accNo: String): SubmissionProjection
```

Comment:
1. Maybe we should call this method `getSubmissionProjection` in order to make it match the response object
2. There's this method called [findLatestBasicByAccNo](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoMetaQueryService.kt#L28) which is only being used [here](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-security/src/main/kotlin/ebi/ac/uk/security/service/UserPrivilegesService.kt#L63) to get the owner. Maybe we could add that field to this method and get rid of that other call which is very inefficient because it loads all the submission just to get the owner

Reply:
sorry not following are you suggesting me to change this other operation to be efficient?

Reply:
personally I do not think is convinient to change a security method that is not related at all with this feautre set of changes, I am pro improve the code that you are changing but IMO changing owner is way beyond that.

 Despite this I did try to refactor for merging but this become problematic as basic submission is used other places and also Project is only part of mongo and not comon persistence, result of this I end up having many changes which IMO are not good to be included in this PR.

Reply:
I understand. If you think this is a valuable change, should we create a ticket for it? Otherwise, let's just let it be 😁